deviations/templates/deviations/add_dl.html

```diff
@@ -1,28 +1,29 @@
-{% extends "edit_course/edit_course_base.html" %}
+{% extends "course/course_base.html" %}
```

Comment:
I made a lot of small changes to the templates because the deviation pages are no longer in the Edit Course menu but in the sidebar. Different base template, different view tags, edittitle -> title, editbreadcrumblist -> breadcrumblist, coursecontent -> columns.